extensions/console/02_console.js

```diff
@@ -599,7 +599,7 @@
           }
           const emptyItems = i - index;
           const ending = emptyItems > 1 ? "s" : "";
-          return dim(`<${emptyItems} empty item${ending}>`);
+          return gray(`<${emptyItems} empty item${ending}>`);
```

Comment:
I think it's better that we just don't use `dim` since it doesn't work in cmd and instead use `gray`, so I've changed this as well. This is the only other occurence of using `dim` in the code.